test/unit/DependencyFactory/SwiftMailerFactoryTest.php

```diff
@@ -19,20 +22,43 @@ function () {
                     \Swift_Mailer::class,
                     $this->assertDefinesService('swiftmailer.mailer', SwiftMailerFactory::definitions())
                 );
+                $this->assertInstanceOf(
```

Comment:
Better to make these separate tests, if the first assertion fails with the optional dependency thing then it will be caught by the outer ->assertOptionalService and none of the other asserts in the block will run.